sklearn/decomposition/pca.py

```diff
@@ -138,7 +142,40 @@ class PCA(BaseEstimator, TransformerMixin):
         Whitening will remove some information from the transformed signal
         (the relative variance scales of the components) but can sometime
         improve the predictive accuracy of the downstream estimators by
-        making there data respect some hard-wired assumptions.
+        making their data respect some hard-wired assumptions.
+
+    svd_solver : string {'auto', 'full', 'arpack', 'randomized'}
+        auto :
+            the solver is selected by a default policy based on X.shape and
+            n_components.
+        full :
+            run exact SVD calling ARPACK solver via
+            `scipy.linalg.svd` and select the components by postprocessing
+        arpack :
```

Comment:
Was this solver just introduced for completeness sake or is there some advantage of using it?